src/Forms/__tests__/ValidationErrorMessage.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  render,
+  Matcher,
+  NormalizerFn,
+} from 'test-utils';
+import ValidationErrorMessage from '../ValidationErrorMessage';
+
+type ReactGetByText = (
+  text: Matcher,
+  options?: {
+    selector?: string;
+    exact?: boolean;
+    ignore?: string|boolean;
+    normalizer?: NormalizerFn;
+  }) => HTMLElement;
+
```

Comment:
It looks like `@testing-library/react` does have [a `@types` package](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/testing-library__react/index.d.ts) we can use for this. There is a `GetByText` type that's defined as part of [the `@testing-library/dom` typings](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/testing-library__dom/queries.d.ts), but to use it with the react package you'd have to type it as:

```ts
let getByText: BoundFunction<GetByText>;
```

That's still a bit cumbersome, but we could (in a separate ticket) look at re-exporting all of the queries as `BoundFunction<>` in our `test-utils` file so we can more easily type all of these in the future.

Reply:
Ah thank you! I'm bookmarking this. Fixed through 8079a76.